nav_msgs/Path ref_path
bool terminate
---
float64 linear_acc
float64 steer_angle
